### DISCUSSION TITLE
[bank tags] automatic placeholder substitution of alternative versions of item

### DISCUSSION THREAD

**Discussion by an0n716** [runelite/runelite#15993] [Ideas]
Would be nice to have an option you could toggle on each item placeholder, _individually_ (as you might not want it for all placeholders of that item), that tells bank tags to automatically swap the item placeholder, in the case the item is unavailable, with any of its item variants, if they **are** available. 

This would avoid the inconvenience of having to move your degraded items back into the bank tag and then release the old one, every time they degrade. 

I imagine this could work for all degradable items, as well as things such as enhanced item versions (berserker ring -> berserker ring (i) for example). 

Perhaps an editable list could be exposed in the settings for people to change the prioritization to their liking, something like this:
```
Verac's plateskirt, Verac's plateskirt 100, Verac's plateskirt 90, ...
Berserker ring (i), Berserker ring, Warrior ring (i), Warrior ring, Ring of wealth
```
The bottom line is my idea of how it could alternatively be used, but it could just as well be restricted to only items of the same type.

**Comment by an0n716** [comment]
actually on further inspection this seem to already partially be implemented
when you drag an item into a bank tag which already contain an empty placeholder for an alternative version of that item, it overwrites it.
Its just not automatic

**Comment by Alexsuperfly** [comment]
thats this #11501

**Comment by Alexsuperfly** [comment]
the second line you have shown is very subjective so is not something thats available
